python/tskit/tables.py

```diff
@@ -2601,6 +2601,24 @@ def sort(self, edge_start=0):
         self._ll_tables.sort(edge_start)
         # TODO add provenance
 
+    def canonicalise(self):
+        """
+        This puts the tables in *canonical* order - to do this, the individual
```

Comment:
I think it's probably worth a section in the data-model docs to describe what the canonical form is. We can then just refer to that here to describe what this is doing.